.github/workflows/images-legacy.yaml

```diff
@@ -61,6 +61,7 @@ jobs:
           file: ${{ matrix.dockerfile }}
           push: true
           tags: |
+            quay.io/${{ github.repository_owner }}/${{ matrix.name }}-ci:latest
```

Comment:
How do we know this is only for the pushed master branch version? Is it because of `push: true` two lines above?

Reply:
@joestringer no, because we only accept 2 event types 'pull_request_target' and 'push', and there's an if statement on line 56 that runs this step if it is not 'pull_request_target'.